views/applicationForms/usersBasicDetails.ejs

```diff
@@ -116,14 +118,17 @@
                     </span>
 
       <input type="tel" id="mobileNo" class="govuk-input govuk-!-width-one-half"  autocomplete="on" name="mobileNo"  <% if (submit_status===true) { %>disabled="disabled"<% } %> value="<% if(form_values && form_values.mobileNo!=null){%><%=form_values.mobileNo.replace(/\s/g, "")%><%} %>">
-    </fieldset>
   </div>
     </fieldset>
 
-    <fieldset class="inline" style="margin-bottom: 20px">
+    <fieldset class="govuk-fieldset">
```

Comment:
You might need to re-add in the specific styling for this fieldset.

Reply:
done